api/hex-bot.ts

```diff
@@ -1,12 +1,109 @@
 import { NowRequest, NowResponse } from '@now/node';
 import { Octokit } from '@octokit/rest';
-import Telegraf, { ContextMessageUpdate, Extra } from 'telegraf';
+import Telegraf, { ContextMessageUpdate, Extra} from 'telegraf';
 import { ExtraEditMessage } from 'telegraf/typings/telegram-types';
-import axios from 'axios';
+// import axios from 'axios';
```

Comment:
you sure you dont need axios ... i even dont see the calls to probot server... maybe you would add later and you didn't want the annoying error of nodemon saying never used the axios declaration..anywho..bye

Reply:
I hadn't added that part yet.